server/pulp/server/webservices/views/util.py

```diff
@@ -114,6 +114,21 @@ def wrapper(*args, **kwargs):
 json_body_allow_empty = functools.partial(json_body_required, allow_empty=True)
 
 
+def json_type_required(json_type):
```

Comment:
need a doc block. See the one below for an example. It needs to have a brief description of what this function is supposed to do or why it exists, then describe the argument and declare its type.